openml/runs/functions.py

```diff
@@ -391,26 +395,103 @@ def _run_task_get_arffcontent(
     # TODO use different iterator to only provide a single iterator (less
     # methods, less maintenance, less confusion)
     num_reps, num_folds, num_samples = task.get_split_dimensions()
+    classes = None
 
+    n_fit = 0
     for rep_no in range(num_reps):
         for fold_no in range(num_folds):
             for sample_no in range(num_samples):
+                n_fit += 1
```

Comment:
I don't see a reason not to use `enumerate` (it looks like you want to start at 1, which is done with `enumerate(Sequence, start=1)`

Reply:
Done, thanks for the tip with `start=1`.